### PR TITLE
[Plans] Handle UTM params in URL

### PR DESCRIPTION
**Pull request by sorta**
For https://travisci.assembla.com/spaces/Web/tickets/220/details

### CODE REVIEW

File: app/utils/object-collect.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      const item = this.get(key);
+      if (isPresent(item)) {
+        collection[key] = item;
+      }
+      return collection;
+    }, {});
+
+    return result;
+  });
+}
```
Comment (piotr-travisci) [inline app/utils/object-collect.js]: Nice tool 👍

**Comment by sorta** [comment]
Thanks for the review @piotr-travisci & @artursmirnov! :heart:

RE the snake casing: yeah, I don't particularly like that part of it. One alternative could be to gather the queryParams together into `@utmFields` in the controller. I thought that might be weird, but that was before I abstracted things a bit more into the `lead` model and the `objectCollect` util. Idk, what do you think?

File: app/utils/object-collect.js (hunk 1 above)
Comment (sorta) [reply app/utils/object-collect.js]: Thanks! :smile:

**Comment by artursmirnov** [comment]
@sorta I really like how simple it is now, and I don't think that case in naming worth code complication. Let's just keep it as it is - it's so good 👍

**Comment by sorta** [comment]
Ok yeah, that sounds good to me :+1:  Thanks!

**Pull request merged**